src/app/views/schedule/Schedule.scss

```diff
@@ -4,7 +4,9 @@
   text-align: center;
   min-height: 100vh;
   min-width: 100vw;
-  background: linear-gradient(270deg, #191970 0%, #a23fd1 100%);
+  background: url(../../../assets/images/background_terrain.png);
+	background-size: cover;
+	background-repeat: no-repeat;
```

Comment:
Rather than repeating ourselves, could we create a [mixin](https://sass-lang.com/documentation/at-rules/mixin) for the terrain background that's included in the Home page and Schedule page?

Reply:
Yes (I didn't see a specific file with other mixins, so I added it to globals/zothacks-helper.scss). Changes made [here](https://github.com/HackAtUCI/ZotHacks-Site/pull/51/commits/f2489686aea0495e5b1112394fcc66378a8681ce).